modules/coreos-layering-configuring-on-proc.adoc

```diff
@@ -130,6 +130,37 @@ machine-os-builder-6fb66cfb99-zcpvq                                     1/1
 <1> This is the build pod where the custom layered image is building, named in the `build-<MachineOSConfig_CR_name>-<hash>` format.
 <2> This pod can be used for troubleshooting.
 
+. Verify custom layered image build by running the a command similar to the following:
++
+[source,terminal]
+----
+$ oc get machineconfigpool <mcp_name> -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  labels:
+    machineconfiguration.openshift.io/mco-built-in: ""
+    pools.operator.machineconfiguration.openshift.io/layered: ""
+  name: layered
+# ...
+status:
+# ...
+  conditions
+# ...
+  - lastTransitionTime: "2025-09-09T13:43:35Z"
+    message: 'Failed to build OS image for pool worker (MachineOSBuild: worker-2d03dc921ff0c242c5892a3ef1ed1608):
+      Failed: Build Failed'
+    reason: BuildFailed
+    status: "False" <1>
```

Comment:
Since the message here is showing the build Failed, the Status here should show `True` and not False.